src/tests/unit/tests/electron/views/automated-checks/automated-checks-view.test.tsx

```diff
@@ -32,15 +32,15 @@ describe('AutomatedChecksView', () => {
 
         const scanStatues = [undefined, ScanStatus[ScanStatus.Scanning], ScanStatus[ScanStatus.Failed]];
 
-        it.each(scanStatues)('status scan <%s>', scanStatusName => {
+        it.each(scanStatues)('when status scan <%s>', scanStatusName => {
```

Comment:
`scanStatues` looks like a typo. Should it be `scanStatuses` ?

Reply:
uh... Code Spell Checker extension didn't flag this

Reply:
updated